src/Service/Caller/Masa.Contrib.Service.Caller.HttpClient/README.zh-CN.md

```diff
@@ -0,0 +1,111 @@
+中 | [EN](README.md)
+
+## Masa.Contrib.Service.Caller.HttpClient
+
+## 用例:
+
+```c#
+Install-Package Masa.Contrib.Service.Caller
+Install-Package Masa.Contrib.Service.Caller.HttpClient
+```
+
+### 基本用法:
+
+1. 修改`Program.cs`
+
+    ``` C#
+    builder.Services.AddCaller(options =>
+    {
+        options.UseHttpClient(httpClientBuilder =>
+        {
+            httpClientBuilder.Name = "UserCaller";// 当前Caller的别名，仅存在一个HttpClient时，可以不对Name赋值
+            httpClientBuilder.BaseAddress = "http://localhost:5000" ;
+        });
+    });
+    ```
+
+2. 如何使用:
+
+    ``` C#
+    app.MapGet("/Test/User/Hello", ([FromServices] ICallerProvider userCallerProvider, string name)
```

Comment:
userCallerProvider?

Reply:
This will be changed to userCaller, reflecting the purpose of User is to inform the user that the current Caller is the Caller where `http://localhost:5000` is located